app/webpack.config.dev.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+module.exports = {
+    mode: 'development',
+    output: {
+        filename: '[name].bundle.js'
+    },
+    devServer: {
+        contentBase: path.join(__dirname, 'dist'),
+        port: 4200
+    },
+    devtool: 'source-map',
+    target: 'web',
```

Comment:
u can omit it as well. it's by default

Reply:
It's a little bit strange, but when I remove `target: 'web'`, hot reloading doesn't work!